acct-group/minbif/metadata.xml

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
```

Comment:
don't remove any of the `acct-*` packages yet, the policy and discussions around these aren't finalised yet

Reply:
Okay, I've removed acct packages